### PR TITLE
ko: Remove invisibles

### PR DESCRIPTION
**Pull request by SnowMarble**
part of #11737 for ko

### CODE REVIEW

**Review by hochan222 — APPROVED** [review]
@SnowMarble 님, 오.. 이런 문제가 있다니 신기하네요. 혹시 어떻게 찾아내신건지 여쭤봐도 괜찮을까요?

**Comment by SnowMarble** [comment]
@hochan222  vscode에서 regex로 검색했더니 찾아지더라고요ㅎ

File: files/ko/web/javascript/reference/errors/illegal_character/index.md [hunk 1]
```diff
@@ -68,14 +68,14 @@ var colors = ['#000', '#333', '#666'];
 다른 곳에서 코드를 복사하여 붙여넣은 경우 잘못된 문자가 포함돼 있을 수 있습니다. 주의하세요!
 
 ```js example-bad
-var foo = 'bar';​
+var foo = 'bar';
```
Comment (queengooborg) [inline files/ko/web/javascript/reference/errors/illegal_character/index.md:71]: This line should actually be left alone; it's an example regarding the zero-width space character (0x200B) itself!

Comment (SnowMarble) [reply files/ko/web/javascript/reference/errors/illegal_character/index.md:71]: oh, I should have been more careful. thanks!

Comment (queengooborg) [reply files/ko/web/javascript/reference/errors/illegal_character/index.md:71]: All good; I almost missed it in the Spanish locale as well when doing a search-and-replace, haha!

**Pull request merged**